pkg/gadgets/bindsnoop/types/types.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2019-2022 The Inspektor Gadget authors
+
+package types
+
+import (
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+)
+
+type Event struct {
+	eventtypes.Event
+
+	Pid       uint32 `json:"pid,omitempty"`
+	Comm      string `json:"comm,omitempty"`
+	Protocol  string `json:"proto,omitempty"`
+	Addr      string `json:"addr,omitempty"`
+	Port      uint16 `json:"port,omitempty"`
+	Options   string `json:"opts,omitempty"`
```

Comment:
I'm not sure about it. It's true that a string makes it easier to our client, but I think other clients will have extra work to parse this string.

Could it be a `map[string]bool`, for each flag we set `true/false`?

Reply:
I first implemented it as a `map[string]bool`.
But on the CLI, the conversion of the map to simple string was unordered (because I think I cannot do any assumption on the key order of the map).
I can still order it again in the CLI but here I preferred to stick with simple `string`.

Reply:
OK, let's use a string then.